dhcp6c.c

```diff
@@ -176,6 +177,9 @@ main(int argc, char *argv[])
 		case 'i':
 			infreq_mode = 1;
 			break;
+		case 'l':
```

Comment:
I like the fact that we can keep this optional for introduction, but in the grand scheme of things locking the PID file should be enough and thus we don't even need to point to a file.